### ISSUE TITLE
dd: failed to open '/etc/openvpn/pki/.rnd': No such file or directory

### ISSUE THREAD

**Issue by ymping** [wheelybird/openvpn-server-ldap-otp#52]
Start openvpn with command
```bash
docker run \
           --name openvpn-test \
           --volume /home/sysadmin/ovpn:/etc/openvpn \
           --detach=true \
           -e "OVPN_SERVER_CN=myserver.mycompany.com" \
           -e "LDAP_URI=ldap://ldap.mycompany.com" \
           -e "LDAP_BASE_DN=dc=mycompany,dc=com" \
           --cap-add=NET_ADMIN \
           wheelybird/openvpn-ldap-otp:v1.5
```
It show a error `dd: failed to open '/etc/openvpn/pki/.rnd': No such file or directory` and container stoped .
I Fix this by manual touch the directory `pki` (`/home/sysadmin/ovpn/pki`) and restart the container.

Detail error message:
```bash
[sysadmin@openvpn ~]$ docker logs openvpn-test

2022-01-05 08:58:25 - Setting up networking...

iptables: masquerade from 10.50.50.0/24 to everywhere via eth0
Checking for existing iptables rule: iptables: No chain/target/match by that name.

2022-01-05 08:58:25 - Setting up PKI...

easyrsa: creating server certs
dd: failed to open '/etc/openvpn/pki/.rnd': No such file or directory
[sysadmin@openvpn ~]$ 
```

**Comment by wheelybird** [comment]
Hi.  Thanks for reporting this.  I've pushed a fix.  Run `docker pull wheelybird/openvpn-ldap-otp:v1.5` to ensure you've got the updated container image before trying again.

**Comment by ymping** [comment]
Fix by d0aca36a193c0f74190d41e7fcd13e7113d7ef55
